src/vs/base/test/common/uri.test.ts

```diff
@@ -151,15 +150,15 @@ suite('URI', () => {
 		assert.equal(value.path, '/c:/test/me');
 		assert.equal(value.fragment, '');
 		assert.equal(value.query, '');
-		assert.equal(value.fsPath, normalize('c:/test/me', true));
+		assert.equal(value.fsPath, normalize('c:/test/me'));
 
 		value = URI.parse('file://shares/files/c%23/p.cs');
 		assert.equal(value.scheme, 'file');
 		assert.equal(value.authority, 'shares');
 		assert.equal(value.path, '/files/c#/p.cs');
 		assert.equal(value.fragment, '');
 		assert.equal(value.query, '');
-		assert.equal(value.fsPath, normalize('//shares/files/c#/p.cs', true));
+		assert.equal(value.fsPath, normalize('//shares/files/c#/p.cs'));
```

Comment:
On *nix we create UNC-fsPaths like this `//server/share/folder/file.ts`, and the use of normalise breaks that.

Reply:
@jrieken I will need your help to fix the URI tests, no matter how I turn the `path.normalize` (unix, win32) it fails. Thanks.

Reply:
Yeah, it will always fail. It is tricky, the *proper* fix would be change how `fsPath` is generated but I am not brave enough for that change, esp. not before vacations. You can try the ghetto normalise, e.g. replacing \ with / on *nix and / with \ on windows. That's what URI itself is doing